Makefile

```diff
@@ -8,9 +8,8 @@ MAN5=		portshaker.conf.5 portshaker.d.5
 MAN8=		portshaker.8
 
 FILES=		merge-updating.awk portshaker.conf.sample portshaker.subr
-FILESDIR_merge-updating.awk=		${SHAREDIR}/portshaker
+FILESDIR=	${SHAREDIR}/portshaker
 FILESDIR_portshaker.conf.sample=	${ETCDIR}
```

Comment:
Thank your for looking into this.  I am testing these changes and trying to understand how can `FILESDIR_merge-updating.awk` / `FILESDIR_portshaker.subr` cause failure and `FILESDIR_portshaker.conf.sample` be okay (if that's what I understand).

Reply:
It was not `FILESDIR_merge-updating.awk` / `FILESDIR_portshaker.subr` that caused the issue, but rather the lack of `FILESDIR`. I just chose to remove them since they weren't needed when `FILESDIR` was defined to the same directory.

Reply:
Yup, it's what I understood.  Looks like you need to set `FILESDIR` and it used to work without setting an explicit value before.

Thanks for digging in the problem and finding the culprit!